assemblies/assembly_planning/main.adoc

```diff
@@ -110,7 +110,7 @@ A development environment typically consists of one {CENTRAL} installation and a
 A runtime environment consists of one or more {KIE_SERVER} instances with or without {CENTRAL}. {CENTRAL} enables you to manage the containers deployed on the {KIE_SERVERS}. If {CENTRAL} is not installed, you can use the headless {PRODUCT_SHORT} controller instead. Applications can interact with the {KIE_SERVER} through the REST or JMS API, or the {KIE_SERVER} Java client API.
 
 .Clustered environments
-By clustering two or more computers, you have the benefits of high availability, enhanced collaboration, and load balancing. High availability decreases the chance of a loss of data when a single computer fails because data on all of the computers (nodes) of the cluster is replicated on each node of the cluster. When a computer fails, another computer fills the gap by providing a copy of the data that was on the failed computer. When the failed computer comes online again, it resumes its place in the cluster. Load balancing shares the computing load across the nodes of the cluster. Doing this improves the overall performance.
+By clustering two or more computers, you have the benefits of high availability, enhanced collaboration, and load balancing. High availability decreases the chance of a loss of data when a single computer fails. When a computer fails, another computer fills the gap by providing a copy of the data that was on the failed computer. When the failed computer comes online again, it resumes its place in the cluster. Load balancing shares the computing load across the nodes of the cluster. Doing this improves the overall performance.
```

Comment:
I don't think "doing this" is ok with the style guide, at least from what I remember from my IBM days.

Reply:
I don't find any reference to 'doing this' in the style guide.